python/GafferSceneTest/IECoreGLPreviewTest/RendererTest.py

```diff
@@ -81,7 +80,7 @@ def testOtherRendererAttributes( self ) :
 	def testPrimVars( self ) :
 
 		renderer = GafferScene.Private.IECoreScenePreview.Renderer.create( "OpenGL" )
-		renderer.output( "test", IECoreScene.Output( self.temporaryDirectory() + "/testPrimVars.exr", "exr", "rgba", {} ) )
+		renderer.output( "test", IECoreScene.Output( ( self.temporaryDirectory() / "testPrimVars.exr" ).as_posix(), "exr", "rgba", {} ) )
```

Comment:
Is `as_posix()` a hard requirement here? Again, I would have guessed that `str()` would have worked.

Reply:
Fixed in 75510d5a3a3ec430a94c00553f283b6a26cb924b